sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -126,6 +126,7 @@ export interface CreateQueueOptions extends OperationOptions {
 
 // @public
 export interface CreateReceiverOptions<ReceiveModeT extends ReceiveMode> {
+    maxLockAutoRenewDurationInMs?: number;
```

Comment:
`maxLockAutoRenewDurationInMs` -> `maxMessageLockAutoRenewDurationInMs` ?

The `maxLockAutoRenewDurationInMs` makes it sound like the lock renewal is somehow meant for the receiver instead of the message. Thoughts?

Reply:
What do you think about this then:
maxMessageLockRenewalDurationInMs

And just taking the word 'auto' out of there entirely?

Reply:
This is a word I'd like to be consistent on with our .Net and Java friends.
I thought of removing the word "auto" once, but worried that it might conflict with the duration for a single lock renew which is what you set in the portal or use management apis for

Reply:
I will maintain consistency for team unity!

Reply:
It's now `maxAutoLockRenewalDurationInMs` for consistency with .net.

There's a good explanatory comment on there and I don't think we're going to get a lot of cross-talk between sessions and non-sessions so there will probably be no confusion.